content/en/tracing/guide/users-accounts.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Setting and querying user and account information in traces
+disable_toc: false
+further_reading:
+- link: "/real_user_monitoring/correlate_with_other_telemetry/apm/"
+  tag: "Documentation"
+  text: "Connect RUM and APM"
+- link: "/standard-attributes/"
+  tag: "Documentation"
+  text: "Standard Attributes"
+- link: "/product_analytics/profiles/"
+  tag: "Documentation"
+  text: "User and Account Profiles"
+---
+
+## Overview
+
+Getting visibility into users and accounts in APM helps you understand which users are affected by performance issues or errors. User and account information is displayed in the [APM Investigator][8] and [Error Tracking issues][9]. Soon, this information will also be available in [service pages][13], [resource pages][14] and in the [trace panel][15].
+
+{{< img src="tracing/guide/user-accounts/user-account-apm-investigator.png" alt="User and account information displayed in APM Investigator showing impacted users and accounts" style="width:100%;" >}}
+
+By tagging traces with user and account identifiers, you can:
+- Track which users are impacted by backend errors or latency issues
+- Analyze performance split by user or account segments
+- Monitor user-specific behaviors across your distributed systems
+
+## Tag user and account information in spans
+
+### From RUM
+
+If you're already collecting Real User Monitoring data, you can propagate user and account information from browser or mobile applications to your backend traces:
+
+1. Set user and account information using the [`datadogRum.setUser()`][1] and [`datadogRum.setAccount()`][2] APIs in your browser application.
+
+2. Enable trace propagation by configuring `allowedTracingUrls` in your RUM SDK initialization. See [Connect RUM and Traces][3] for detailed setup instructions. Additionally, set `propagateTraceBaggage` to `true`, to automatically propagates user and account context in the [baggage][10] to backend traces alongside the trace context.
+
+The user and account information is automatically remapped in the backend to the [`usr.id` and `account.id` standard attributes][4], making it consistent across all your traces. Standard attributes allow you to filter and search your trace data consistently across all your services.
+
+### From APM SDKs
+
+For backend services or applications without RUM, you can tag spans directly using APM SDKs:
+
+1. Use the [span tagging API][5] (`set_tag`, `SetTag`, or `setTag` depending on your language) to add `usr.id` and `account.id` attributes to your spans.
+
+2. To propagate user and account information across service boundaries, use [trace context propagation with baggage][6].
+
+**Example (Python):**
+
+```python
+from ddtrace import tracer
+
+# Tag the span with user and account information
+span = tracer.current_span()
```

Comment:
maybe you could suggest to use the `tracer.current_root_span()` sdk to make sure the user is properly set on the service entry span